src/main/bash/sdkman-install.sh

```diff
@@ -97,6 +107,7 @@ function __sdkman_install_local_version {
 
 	else
 		__sdkman_echo_red "Invalid path! Refusing to link ${candidate} ${version} to ${folder}."
+		return 1
```

Comment:
Added the `return 1` here as well let me know if you want me to follow up with setting up different exit codes depending on the errors.